keadm/cmd/keadm/app/cmd/beta/init.go

```diff
@@ -105,6 +106,9 @@ func addInitBetaJoinOtherFlags(cmd *cobra.Command, initBetaOpts *types.InitBetaO
 
 	cmd.Flags().StringVar(&initBetaOpts.ExternalHelmRoot, types.ExternalHelmRoot, initBetaOpts.ExternalHelmRoot,
 		"Add external helm root path to keadm.")
+
+	cmd.Flags().StringVar(&initBetaOpts.TarballURL, types.TarballURL, initBetaOpts.TarballURL,
```

Comment:
So we must add the chart to release, right? :)

Reply:
Yes, edgemesh supports the tgz file in its codes, this way is also ok.